kernel/packages/unity-interface/dcl.ts

```diff
@@ -185,82 +174,41 @@ export async function startGlobalScene(
   })
 }
 
-export async function startPortableExperienceScene(unityInterface: UnityInterface, cid: string, peId: string) {
-  const scene = new UnityPortableExperienceScene(await getPortableExperienceFromS3Bucket(cid, peId))
+export async function spawnPortableExperienceScene(portableExperienceUrn: PortableExperienceUrn): Promise<string> {
+  const parsedUrn: DecentralandAssetIdentifier | null = await parseUrn(portableExperienceUrn)
+
+  defaultLogger.info('URN...', parsedUrn)
+
+  if (!parsedUrn || !isPortableExperience(parsedUrn)) {
+    throw new Error(`Could not parse portable experience from urn: ${portableExperienceUrn}`)
+  }
+
+  const scene = new UnityPortableExperienceScene(
+    await getPortableExperienceFromS3Bucket((parsedUrn as unknown) as OffChainAsset)
+  )
   loadParcelScene(scene, undefined, true)
+  const parcelSceneId = getParcelSceneID(scene)
   unityInterface.CreateUIScene({
-    id: getParcelSceneID(scene),
+    id: parcelSceneId,
     name: scene.data.name,
     baseUrl: scene.data.baseUrl,
     contents: scene.data.data.contents,
     icon: scene.data.data.icon,
     isPortableExperience: true
   })
+  return parcelSceneId
 }
 
-export async function getPortableExperienceFromS3Bucket(cid: string, peId: string) {
-  const baseUrl: string = `${STATIC_PORTABLE_SCENES_S3_BUCKET_URL}/${peId}/`
-
-  const mappingsFetch = await fetch(`${baseUrl}mappings`)
-  const mappingsResponse = (await mappingsFetch.json()) as MappingsResponse
-
-  const sceneJsonMapping = mappingsResponse.contents.find(($) => $.file === 'scene.json')
-
-  if (sceneJsonMapping) {
-    const sceneResponse = await fetch(`${baseUrl}${sceneJsonMapping.hash}`)
-
-    if (sceneResponse.ok) {
-      const scene = (await sceneResponse.json()) as SceneJsonData
-      return getLoadablePortableExperience({
-        cid,
-        baseUrl: `${baseUrl}`,
-        mappings: mappingsResponse.contents,
-        sceneJsonData: scene
-      })
-    } else {
-      throw new Error('Could not load scene.json')
-    }
-  } else {
-    throw new Error('Could not load scene.json')
-  }
+function isPortableExperience(dclId: DecentralandAssetIdentifier): boolean {
```

Comment:
Type assertions to the rescue

```suggestion
function isPortableExperience(dclId: DecentralandAssetIdentifier): dclId is OffChainAsset {
```

It makes this `as unknown` no longer necessary

```ts
(parsedUrn as unknown) as OffChainAsset
```